recipes/diffpy.fourigui/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "diffpy.fourigui" %}
+{% set version = "0.1.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://test.pypi.io/packages/source/d/diffpy_fourigui/diffpy_fourigui-{{ version }}.tar.gz
+  sha256: 7f5f7b3941f070766f698e183849ff3c027f289a47e3aff18b48074646767532
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools
+    - setuptools-git-versioning >=2.0
+    - pip
+
+  run:
+    - python >=3.10
+    - h5py
+    - tk
+    - matplotlib-base
+    - numpy
+
+test: 
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - conda list
+    - pip list
```

Comment:
I believe you don't need this. The list of packages is in the log and this is not a test of the package.

```suggestion
```

Reply:
Yes, removed.